frontend/src/components/chat/ChatBody.js

```diff
@@ -20,11 +20,11 @@ const ChatBody = ({ chatSocket, username, room_id, is_live }) => {
   })
 
   chatSocket.on('message response', (data) => {
```

Comment:
I have actually moved the .on listeners to "RoomPage.js" to allow Chat to work with the "Popover" MUI component. No worries the code isn't too different.

Reply:
Nw, we can wait for the other pr to be merged in then ill fix the differences

Reply:
Thank you a lot, sorry for inconveniences on your own